packages/core/src/routes/custom-phrase.ts

```diff
@@ -23,4 +23,23 @@ export default function phraseRoutes<T extends AuthedRouter>(router: T) {
       return next();
     }
   );
+
+  router.put(
+    '/custom-phrases/:languageKey',
+    koaGuard({
+      params: CustomPhrases.createGuard.pick({ languageKey: true }),
+      body: translationGuard,
+      response: CustomPhrases.guard,
```

Comment:
just curious, when should we guard the response when should not?

Reply:
- I think it's better guard 
- The response guard will be used to generate the response part of `swagger.json` in the future.